spec/commands/v2/patch_link_set_spec.rb

```diff
@@ -10,7 +10,7 @@
     {
       content_id: content_id,
       links: {
-        topics: topics,
+        topics: topics * 2, # test deduplication
```

Comment:
Pretty cool that the behaviour and updated test case for it all sum up to 9 bytes (`.uniq` and ` * 2`).